### DISCUSSION TITLE
Lost/Cannot renew GitHub student developer pack status

### DISCUSSION THREAD

**Discussion by shenbenson** [community/community#46899] [GitHub Education]
I initially applied and got approved for the GitHub student developer pack in 2020. Recently I noticed that I lost access to GitHub Pro and no longer have the developer pack. I tried to submit another application but I got rejected by the system (automatic review) even though all my documents are valid and I am a current student.

Please help.

**Comment by chris-c-thomas** [comment]
That's unfortunate. By any chance do any of these [reasons](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved) apply to you? Graduating this academic year, maybe?

Otherwise you can try reaching out to support regarding education here: https://support.github.com/request/education

**Reply by shenbenson** [reply]
I have no idea what's going on anymore. I am 100% certain I qualify and I provided support documents. Also, I am not graduating until 2025, so that's not a problem.

I just opened a ticket through the support link you provided and it got IMMEDIATELY CLOSED by what appears to be the GitHub Support bot...

**Comment by samus-aran** [comment]
Hi @shenbenson  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
